src/display/api.js

```diff
@@ -145,6 +145,11 @@ function setPDFNetworkStreamFactory(pdfNetworkStreamFactory) {
  *   converted to OpenType fonts and loaded via font face rules. If disabled,
  *   fonts will be rendered using a built-in font renderer that constructs the
  *   glyphs with primitive path commands. The default value is `false`.
+ * @property {boolean} [fontExtraProperties] - Include additional properties,
+ *   which is unused during rendering of PDF documents, when exporting the
+ *   parsed font data from the worker-thread. This may be useful for custom
```

Comment:
I'm wondering: is the objective to keep support this or is it only meant as a temporary solution so we can check if someone is actually using this and providing a way for them to get the data?

If we want to get rid of this eventually, I'm not sure about the `may be useful for custom implementations` part. Given that these properties were never meant to be exposed, and are of very limited use anyway if someone is actually using them, it feels like we should emphasize here that this option is only provided to for temporary backwards compatibility and that it may be removed in new releases. Otherwise it may read as if we're advertising the usage for third-party implementations.

Reply:
> I'm wondering: is the objective to keep support this or is it only meant as a temporary solution so we can check if someone is actually using this and providing a way for them to get the data?

Good question; I'm not really sure if/when we even can/want to eventually get rid of this.

Note that with debugging enabled in the viewer, e.g. with `#pdfBug=all`, it's currently possible to log all of these properties on the main-thread which occasionally does come in handy while debugging font issues.

> If we want to get rid of this eventually, I'm not sure about the `may be useful for custom implementations` part.
> [...]
> Otherwise it may read as if we're advertising the usage for third-party implementations.

You're right, let's at least remove that part to not unnecessarily promoting it.